src/Introspection/IntrospectionTest.cpp

```diff
@@ -1,7 +1,8 @@
-// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
```

Comment:
mm? I do not think we want to do it on move...

Reply:
The internal guidelines say that changing this on modification is optional (because it would be annoying). But I don't know why I did it so let me just change it back.